release.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
`-e`

otherwise it keeps going on failures.

Reply:
Added `-euxo pipefail` on the second line! ;)